include/word.h

```diff
@@ -19,15 +18,61 @@
 bool valid_word(dict_t *dict, char* word);
 
 /*
- * generate_suggestions - Returns a list of possible suggestions
+ * Helper function for generate_suggestions()
+ *
+ * word_check_cap - Checks if a word has a type of capitalization.
+ *
+ * Parameters:
+ *  - word: A string (word).
+ *
+ * Returns:
+ *  - -1: Error - Not a word.
+ *  - 0: Not capitalized.
+ *  - 1: First letter is capitalized.
+ *  - 2: Every letter is capitalized.
+ *  - 3: Inconsistent capitalization.
+ */
+int word_check_cap(char *word);
+
+/*
+ * Helper function for generate_suggestions()
+ *
+ * word_lowercase - Gets a string (word) and lowercases it.
  *
  * Parameters:
  *  - word: A string (word).
+ *
+ * Returns:
+ *  - char *: Returns the lowercased version of the string (word).
+ */
+char *word_lowercase(char *word);
+
+/*
+ * Helper function for generate_suggestions()
+ *
+ * words_uppercase - Gets a string (word) array and uppercases each string (word).
```

Comment:
This needs to be more descriptively named. And why does it take a capitalization state?

Reply:
I'll change it to word_restore_cap, meaning that capitalization of the initial word is restored here.

Reply:
See above, responding to your first comment for clarification of what this does.